dist/vue-masonry-plugin-umd.js

```diff
@@ -3,4 +3,38 @@
+/*!
+ * Outlayer v2.1.1
+ * the brains and guts of a layout library
+ * MIT license
+ */
+/*!
+ * Outlayer v2.1.1
```

Comment:
Oh dear, it seems that the uglify config that was migrated doesn't remove comments for builds. I'll look into how to fix that